pytest_mozwebqa/cloud/saucelabs.py

```diff
@@ -7,124 +7,138 @@
-            capabilities['tags'] = tags
-        if options.platform is not None:
-            capabilities['platform'] = options.platform
-        if options.browser_version is not None:
-            capabilities['version'] = options.browser_version
-
-        executor = 'http://%s:%s@ondemand.saucelabs.com:80/wd/hub' % (
-            self.username, self.api_key)
-        return webdriver.Remote(command_executor=executor,
-                                desired_capabilities=capabilities)
-
-    def url(self, session):
-        return 'http://saucelabs.com/jobs/%s' % session
-
-    def additional_html(self, session):
-        return [self._video_html(session)]
-
-    def update_status(self, session, passed):
-        status = {'passed': passed}
-        requests.put('http://saucelabs.com//rest/v1/%s/jobs/%s' % (self.username, session),
-                     data=json.dumps(status), auth=(self.username, self.api_key))
-
-    def _video_html(self, session):
-        flash_vars = 'config={\
-            "clip":{\
-                "url":"https://assets.saucelabs.com/jobs/%(session)s/video.flv",\
-                "provider":"streamer",\
-                "autoPlay":false,\
-                "autoBuffering":true},\
-            "play": {\
-                "opacity":1,\
-                "label":null,\
-                "replayLabel":null},\
-            "plugins":{\
-                "streamer":{\
-                    "url":"https://cdn1.saucelabs.com/sauce_skin_deprecated/lib/flowplayer/flowplayer.pseudostreaming-3.2.13.swf",\
-                    "queryString":"%%3Fstart%%3D%%24%%7Bstart%%7D"},\
-                "controls":{\
-                    "mute":false,\
-                    "volume":false,\
-                    "backgroundColor":"rgba(0,0,0,0.7)"}},\
-            "playerId":"player%(session)s",\
-            "playlist":[{\
-                "url":"https://assets.saucelabs.com/jobs/%(session)s/video.flv",\
-                "provider":"streamer",\
-                "autoPlay":false,\
-                "autoBuffering":true}]}' % {'session': session}
-
-        return html.div(html.object(
-            html.param(value='true', name='allowfullscreen'),
-            html.param(value='always', name='allowscriptaccess'),
-            html.param(value='high', name='quality'),
-            html.param(value='#000000', name='bgcolor'),
-            html.param(
-                value=flash_vars.replace(' ', ''),
-                name='flashvars'),
-            width='100%',
-            height='100%',
-            type='application/x-shockwave-flash',
-            data='https://cdn1.saucelabs.com/sauce_skin_deprecated/lib/flowplayer/flowplayer-3.2.17.swf',
-            name='player_api',
-            id='player_api'),
-            id='player%s' % session,
-            style='border:1px solid #e6e6e6; float:right; height:240px;'
-                  'margin-left:5px; overflow:hidden; width:320px')
+name = 'Sauce Labs'
+
+
+def _get_config(option):
```

Comment:
I know we discussed this in an earlier PR, but I'll mention it again. This `_get_config()` method is identical to `_get_config()` in `browserstack.py`, except for the `section` being different. I think this duplication could be addressed without making the code more confusing. Perhaps a separate `Config` class, which is used by both?